modules/assim.batch/R/pda.utils.R

```diff
@@ -740,36 +740,25 @@ pda.settings.bt <- function(settings){
   sampler = settings$assim.batch$bt.settings$sampler 
   
   iterations = as.numeric(settings$assim.batch$bt.settings$iter)
-  optimize = settings$assim.batch$bt.settings$optimize
-  if(!is.null(settings$assim.batch$bt.settings$consoleUpdates)) consoleUpdates = as.numeric(settings$assim.batch$bt.settings$consoleUpdates) else consoleUpdates = NULL
-  parallel = settings$assim.batch$bt.settings$parallel
-  adapt = settings$assim.batch$bt.settings$adapt
-  if(!is.null(settings$assim.batch$bt.settings$adaptationInverval)) adaptationInverval = as.numeric(settings$assim.batch$bt.settings$adaptationInverval) else adaptationInverval=NULL
-  if(!is.null(settings$assim.batch$bt.settings$adaptationNotBefore)) adaptationNotBefore = as.numeric(settings$assim.batch$bt.settings$adaptationNotBefore) else adaptationNotBefore=NULL
-  initialParticles=list("prior",as.numeric(settings$assim.batch$bt.settings$n.initialParticles))
+  if(!is.null(settings$assim.batch$bt.settings$optimize)) optimize = settings$assim.batch$bt.settings$optimize else optimize=TRUE
+  if(!is.null(settings$assim.batch$bt.settings$consoleUpdates)) consoleUpdates = as.numeric(settings$assim.batch$bt.settings$consoleUpdates) else consoleUpdates = max(round(iterations/10),100)
+  if(!is.null(settings$assim.batch$bt.settings$adapt)) adapt = settings$assim.batch$bt.settings$adapt else adapt=TRUE
+  if(!is.null(settings$assim.batch$bt.settings$adaptationInverval)) adaptationInverval = as.numeric(settings$assim.batch$bt.settings$adaptationInverval) else adaptationInverval=max(round(iterations/100*5),100)
+  if(!is.null(settings$assim.batch$bt.settings$adaptationNotBefore)) adaptationNotBefore = as.numeric(settings$assim.batch$bt.settings$adaptationNotBefore) else adaptationNotBefore=adaptationInverval
```

Comment:
good point. camelCase vs dot.case is indeed due to different coding styles, the former is the style they use in the package, the latter is what we use in PEcAn. The rest is my bad. I should add the the shortcut for `<-` on my keyboard at the desktop too :)

@dlebauer that I'll do! this bit started as only `if`s then I added the `else`s... I'll change it